src/reactor.js

```diff
@@ -27,6 +28,7 @@ class Reactor {
     const baseOptions = debug ? DEBUG_OPTIONS : PROD_OPTIONS
     const initialReactorState = new ReactorState({
       debug: debug,
+      cache: config.cache || DefaultCache(),
```

Comment:
> How do you envision using LRU cache?

As an opt-in configuration change that is passed when Reactor is created.

> If so how is it exposed to end user?

Probably punt that to @jordangarcia for best way to expose